From 7b3c2014209018b13f96af28bc8406f1633924a4 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Sun, 29 Mar 2015 21:37:21 +0300
Subject: [PATCH] add some symbol keys to english qwerty layout

now, it should be easier (less clicks) to type in email
addresses and urls
---
 system/keyboardlayouts/english.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/system/keyboardlayouts/english.xml b/system/keyboardlayouts/english.xml
index eb4ca62..71ab1a9 100644
--- a/system/keyboardlayouts/english.xml
+++ b/system/keyboardlayouts/english.xml
@@ -8,14 +8,14 @@ Default font lacks support for all characters
     <keyboard>
       <row>1234567890</row>
       <row>qwertyuiop</row>
-      <row>asdfghjkl</row>
-      <row>zxcvbnm</row>
+      <row>asdfghjkl:</row>
+      <row>zxcvbnm./@</row>
     </keyboard>
     <keyboard modifiers="shift">
       <row>1234567890</row>
       <row>QWERTYUIOP</row>
-      <row>ASDFGHJKL</row>
-      <row>ZXCVBNM</row>
+      <row>ASDFGHJKL:</row>
+      <row>ZXCVBNM./@</row>
     </keyboard>
     <keyboard modifiers="symbol,shift+symbol">
       <row>)!@#$%^&amp;*(</row>
-- 
2.1.4

